README.md

```diff
@@ -18,16 +18,18 @@ u-root embodies four different projects.
     [cmds/core](cmds/core) for most of these.
 
 *   A way to compile many Go programs into a single binary with
-    [busybox mode](https://github.com/u-root/gobusybox/#readme).
+    [busybox mode](#build-modes).
 
-*   A way to create initramfs (an archive of files) to use with Linux kernels.
+*   A way to create initramfs (an archive of files) to use with Linux kernels,
+    [embeddable into firmware](#build-an-embeddable-u-root).
 
 *   Go bootloaders that use `kexec` to boot Linux or multiboot kernels such as
     ESXi, Xen, or tboot. They are meant to be used with
     [LinuxBoot](https://www.linuxboot.org). With that, parsers for
     [GRUB config files](pkg/boot/grub) or
     [syslinux config files](pkg/boot/syslinux) are to make transition to
     LinuxBoot easier.
+    The [SystemBoot](#systemboot) command set offers local and network booting.
```

Comment:
Isn't this part of the previous bullet? (I also think we really need to clean up systemboot)

Reply:
Perhaps the previous bullet should be advertising the `boot` and `pxeboot` commands, really

Reply:
Hmm, or just remove the elaboration from the bullet point and put the link to the SystemBoot section behind `[Go bootloaders](#systemboot)`. The section is more suitable for details. WDYT?